tests/plugins/pastefromword/manual/pastetable.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: bug, 4.11.0, 1490, pastefromword
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, pastefromword
```

Comment:
Table from Excel is pastes as plain text. There are several missing plugins (e.g. `table`).